### ISSUE TITLE
Can't import FeatureEngineer, data_split, backtest_stats, backtest_plot...

### ISSUE THREAD

**Issue by chenzc0101** [AI4Finance-Foundation/FinRL#871]
When I want to use FeatureEngineer, data_split, backtest_stats, backtest_plot...

I run my code as usual:

```
from finrl.meta.preprocessor.preprocessors import FeatureEngineer, data_split

from finrl.plot import backtest_stats, backtest_plot, get_daily_return, get_baseline,convert_daily_return_to_pyfolio_ts
```

However, the error occured as following:

`ImportError: cannot import name 'Arguments' from 'elegantrl.train.run' (/usr/local/lib/python3.8/dist-packages/elegantrl/train/run.py)`

I think this bug comes from the recent update on elegantrl.

Can you help me to find another method to use these functions or fix the bug?

**Comment by cedricfarinazzo** [comment]
Same issue #870 
I made a PR to fix this #872

**Comment by chenzc0101** [comment]
> Same issue #870 I made a PR to fix this #872

Thanks @cedricfarinazzo , I think your commit will fix this problem. 

Hi @zhumingpassional , could you please check @cedricfarinazzo 's commit and update it? Thanks.

**Comment by zhumingpassional** [comment]
thanks.
the PR is merged
